fortitude/src/rules/fortitude/allow_comments.rs

```diff
@@ -58,10 +58,8 @@ pub(crate) struct RedirectedAllowComment {
 impl AlwaysFixableViolation for RedirectedAllowComment {
     #[derive_message_formats]
     fn message(&self) -> String {
-        let Self {
-            original, new_code, ..
-        } = self;
-        format!("`{original}` is a redirect to `{new_code}`")
+        let Self { original, .. } = self;
+        format!("`{original}` has been redirected")
```

Comment:
Have we dropped this because we don't know (at this point, at least) if it was a code or a name that's triggered the redirect?

The fix will include the correct one, so this isn't so important, but how difficult would it be to get the redirect here too?

Reply:
Whoops, I changed that on my first pass at fixing this up and forgot to come back to it. It'll be easy enough to fix.

Reply:
Fixed in https://github.com/PlasmaFAIR/fortitude/pull/354/commits/e5ece1e2745b174b902c4452ff69c26a3a86a171